spec/services/partner_submission_service_spec.rb

```diff
@@ -24,7 +24,7 @@
     stub_gravity_partner_communications
     stub_gravity_partner_contacts
     allow(Time).to receive(:now).and_return(
-      Time.new(2_017, 9, 27).in_time_zone('UTC')
+      Time.new(2_017, 9, 27, 12).in_time_zone('UTC')
```

Comment:
**question (non-blocking):** Will an engineer working in the [UTC+13:00](https://en.wikipedia.org/wiki/UTC%2B13:00) or [UTC:14:00](https://en.wikipedia.org/wiki/UTC%2B14:00) timezones encounter the same test failures that you experienced?

Reply:
The problem will be solved for those who are in the +12 time zone (they will have 27 Sept 00:00 in the test), but not for those who are in +13. And it will appear for those who are in the -12 time zone (but I think the number of such people is less than in the [+1...+12] time zone)